Jefferson/1-Fatorial/fatorial.js

```diff
@@ -0,0 +1,11 @@
+console.log("Resultado Fatorial: " + calcFatorial(process.argv[2]));
```

Comment:
What do you think to create a method getNumber() to abstract process.argv[2], so you can be more readable!